apps/version.c

```diff
@@ -37,16 +37,18 @@ const OPTIONS version_options[] = {
     {"r", OPT_R, '-', "Show random seeding options"},
     {"v", OPT_V, '-', "Show library version"},
     {"c", OPT_C, '-', "Show CPU settings info"},
+    {"w", OPT_W, '-', "Show Windows install context"},
```

Comment:
Does this option make any sense on non-windows platforms?

Reply:
No, I'll gate it on __WIN32